AuthService/src/main/resources/application.properties

```diff
@@ -8,10 +8,13 @@ spring.security.oauth2.client.registration.google.redirect-uri=${GOOGLE_REDIRECT
 
 secret-key=${SECRET_KEY}
 
-spring.datasource.url=jdbc:mysql://mysql:3306/mydb
+spring.datasource.url=jdbc:mysql://mysql:3306/service
```

Comment:
Shouldn't just be service here

Reply:
Updated endpoint of the database to match docker-compose.yml